framework/packages/abstract-adapter/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "abstract-adapter"
-version = "0.17.2"
+version = "0.17.3"
```

Comment:
Since this is a breaking change we theoretically should bump the minor version. This is important for other projects like the Cavern Protocol adapter. And we publish the packages separately from our contracts.

Reply:
Should I rename branch to v0.18?

Reply:
No cause it's only the version of the packages.